### PR TITLE
adding course roster download as csv, fixing #1

### PR DESCRIPTION
**Pull request by ncbrown1**

### CODE REVIEW

**Review by pconrad — COMMENTED** [review]
I wonder whether this might be the place we change this to "github_username" (to comply with Diba's request).   (Just in the header, not in the database table name or the code.)

**Comment by pconrad** [comment]
Sorry, that previous comment made no sense---user fail on my part.  I thought I was highlighting and commenting on a particular line of code.   I'll try again.

File: app/views/students/index.html.erb [hunk 1]
```diff
@@ -1,48 +1,49 @@
...
-          <td>TBA</td>
-        <% end %>
-        <td><%= link_to 'Show', student %></td>
-        <td><%= link_to 'Edit', edit_student_path(student) %></td>
-        <td><%= link_to 'Destroy', student, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <th>Perm</th>
+        <th>First name</th>
+        <th>Last name</th>
+        <th>Email</th>
+        <th>Username</th>
```
Comment (pconrad) [inline app/views/students/index.html.erb:18]: Ah, this is the right way to make a line comment. 

"I wonder if THIS is the place to make the change from "username" to "github username"

https://anacapa-grader.slack.com/archives/general/p1483658966000182

Comment (ncbrown1) [reply app/views/students/index.html.erb:18]: Good Idea! Check this commit out: be75d564b5f66d0627ff995497bb26173c34c071

**Pull request merged**